2-mandatory/4-space-colonies.js

```diff
@@ -14,8 +14,8 @@
         Logic: Only strings that start with A, and finish with family
   
 */
-
-function getSettlers() {}
+const isAllowedToStay = (name) => name.includes("family") && name[0] === "A";
+const getSettlers = (names) => names.filter(isAllowedToStay);
 
```

Comment:
Another perfect answer Laleh! Nice one!!! 🎉

If you wanted to combine these you could do:
`const getSettlers = (names) => names.filter(name => name.includes("family") && name[0] === "A");`

I am sure you knew this already, but just wanted to leave the comment 🙌